index.html

```diff
@@ -11,6 +11,1053 @@
+          <div class="d-flex flex-row">
+            <img src="https://d.newsweek.com/en/full/1668839/tropical-storm-eta.webp?w=397&h=265&f=16e5107f98ef1180f7ce669fb320d6be" alt="">
+            <h5><a href="#">Tropical Storm Eta Path, Tracker as Hurricane Watch Issued for Florida</a></h5>
+          </div>
+
+
+          <div class="d-flex flex-row">
+            <img src="https://d.newsweek.com/en/full/1668839/tropical-storm-eta.webp?w=397&h=265&f=16e5107f98ef1180f7ce669fb320d6be" alt="">
+            <h5><a href="#">Tropical Storm Eta Path, Tracker as Hurricane Watch Issued for Florida</a></h5>
+          </div>
+
+          <div class="d-flex flex-row">
+            <img src="https://d.newsweek.com/en/full/1668839/tropical-storm-eta.webp?w=397&h=265&f=16e5107f98ef1180f7ce669fb320d6be" alt="">
+            <h5><a href="#">Tropical Storm Eta Path, Tracker as Hurricane Watch Issued for Florida</a></h5>
+          </div>
+
+        </aside>
+
+      </div>
+      
+    </section>
+
+
+    <section class="us">
+      <h2>Sports</h2>
+
+      <div class="row">
+        <div class="us-1 col-md-4">
+          <div class="card">
+            <img class="card-img-top" src="https://d.newsweek.com/en/full/1667850/elizabeth-warren-william-barr-reactions-election-fraud.webp?w=397&h=265&f=3192c3cfcd67676c45c76353c451b8a2" alt="">
+          <h2 class="card-body">Democrats Criticize Barr for OKing Investigations Into Unproven Voter Fraud</h2>
+          </div> 
+        </div>
+
+        <div class="us-1 col-md-4">
+          <div class="card">
+            <img class="card-img-top" src="https://d.newsweek.com/en/full/1667850/elizabeth-warren-william-barr-reactions-election-fraud.webp?w=397&h=265&f=3192c3cfcd67676c45c76353c451b8a2" alt="">
+          <h2 class="card-body">Democrats Criticize Barr for OKing Investigations Into Unproven Voter Fraud</h2>
+          </div> 
+        </div>
+
+        <aside class="col-md-4">
+          <div class="d-flex flex-row">
+            <img src="https://d.newsweek.com/en/full/1668839/tropical-storm-eta.webp?w=397&h=265&f=16e5107f98ef1180f7ce669fb320d6be" alt="">
+            <h5><a href="#">Tropical Storm Eta Path, Tracker as Hurricane Watch Issued for Florida</a></h5>
+          </div>
+
+
+          <div class="d-flex flex-row">
+            <img src="https://d.newsweek.com/en/full/1668839/tropical-storm-eta.webp?w=397&h=265&f=16e5107f98ef1180f7ce669fb320d6be" alt="">
+            <h5><a href="#">Tropical Storm Eta Path, Tracker as Hurricane Watch Issued for Florida</a></h5>
+          </div>
+
+          <div class="d-flex flex-row">
+            <img src="https://d.newsweek.com/en/full/1668839/tropical-storm-eta.webp?w=397&h=265&f=16e5107f98ef1180f7ce669fb320d6be" alt="">
+            <h5><a href="#">Tropical Storm Eta Path, Tracker as Hurricane Watch Issued for Florida</a></h5>
+          </div>
+
+        </aside>
+
+      </div>
+      
+    </section>
+
+  </div>
+
+    
+
+  <div class="row new-sub">
```

Comment:
- [ ] Remove the class 'row' here and give the class `new-sub` the following properties: `display: flex;` and `flex-wrap: wrap;`